### DISCUSSION TITLE
Recommendations for cache

### DISCUSSION THREAD

**Discussion by zstatmanweil** [stac-utils/titiler-pgstac#194] [Q&A]
Wondering if anyone has setup a caching layer with `titiler-pgstac`. I need something as simple as possible because I don't have a large team maintaining it. I see in the titiler documents there is guidance for a Redis cache. Would I have to extend `titiler-pgstac` and add something like that? Or would something like Cloudfront work?

**Comment by zstatmanweil** [comment]
@vincentsarago curious if you have a recommendation for what you have done. Thank you!

**Reply by vincentsarago** [reply]
I would usually use cloudfront on top of the API to cache things, avoiding adding complexity to the python application

**Reply by zstatmanweil** [reply]
Thank you!

**Comment by hrodmn** [comment]
@bransonf you found a good Redis solution for titiler-pgstac in an eoAPI stack - do you mind sharing that here?
